tests/integration-test-bq-create-dataset.wdl

```diff
@@ -0,0 +1,30 @@
+version development
```

Comment:
Can this be part of https://github.com/susom/wdl-kit/blob/main/tests/integration-test-bq.wdl? So that it does the cleanup of created dtasets

Reply:
I did thought of that but that is the full test. I think we should keep them individual testing as much as possible.